.github/workflows/pr-build.yml

```
@@ -0,0 +1,37 @@
+name: Node.js SDK Continuous Build
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    name: Build Node ${{ matrix.node-version }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - macos-latest
+          - ubuntu-latest
+          - windows-latest
+        node-version:
+          - 10.x
+          - 12.x
+          - 14.x
+          - 15.x
+    steps:
+      - name: Checkout Repository
```

I think it makes sense to be verbose here for people starting with Github actions
```suggestion
      - name: Checkout AWS XRay SDK Node Repository @ default branch latest
```